Tasks/NuGetCommandV2/nugetcustom.ts

```diff
@@ -56,9 +57,16 @@ export async function run(nuGetPath: string): Promise<void> {
             tl.debug(`All URL prefixes: ${urlPrefixes}`);
         }
         let authInfo = new auth.NuGetExtendedAuthInfo(new auth.InternalAuthInfo(urlPrefixes, accessToken, useCredProvider, false), []);
+        
+        const useCredProviderV2: boolean = quirks.hasQuirk(NuGetQuirkName.V2CredentialProvider);
         let environmentSettings: ngToolRunner.NuGetEnvironmentSettings = {
-            credProviderFolder: useCredProvider ? path.dirname(credProviderPath) : null,
-            extensionsDisabled: true
+            credProviderFolder: useCredProvider 
```

Comment:
This should be simplified as you'll only need to pass in both `credProviderFolder` as before, and the new `credProviderV2Path`.